packages/angular/pwa/pwa/index.ts

```diff
@@ -146,11 +146,15 @@ export default function (options: PwaOptions): Rule {
     const { title, ...swOptions } = options;
 
     await writeWorkspace(host, workspace);
+    let assetsDir = posix.join(sourcePath, 'assets');
+    if (!host.exists(assetsDir)) {
+      assetsDir = posix.join(sourcePath, 'public');
+    }
```

Comment:
maybe there's a style guide that doesn't prefer ternaries? but if not:

```suggestion
    const assetsPath = host.exists(assetsDir)
      ? 'assets'
      : 'public';
    const assetsDir = posix.join(sourcePath, assetsPath);
```

Reply:
This will not work, since you are using `assetsDir` prior to being defined.